lib/typing/redshift.go

```diff
@@ -44,11 +55,17 @@ func kindToRedShift(kd KindDetails) string {
 		return "INT8"
 	case Struct.Kind:
 		return "SUPER"
-	case String.Kind, Array.Kind:
+	case Array.Kind:
```

Comment:
Nit: should we call this function `kindToRedshift` with the "S" for "shift" lowercased?

Reply:
Yep, thanks